src/hotspot/share/gc/shared/stringdedup/stringDedupProcessor.cpp

```diff
@@ -69,9 +70,9 @@ void StringDedup::Processor::initialize() {
   _processor = new Processor();
   if (UsePerfData && os::is_thread_cpu_time_supported()) {
     EXCEPTION_MARK;
```

Comment:
This whole `if` block could be updated to `CPUTimeCounters::get_instance()->create_counter(CPUTimeGroups::conc_dedup)`. We can also remove the `_concurrent_dedup_thread_cpu_time` field and the `ThreadTotalCPUTimeClosure(PerfCounter* counter)` constructor.

In `StringDedup::Processor::run()`, it can call
```
if (UsePerfData && os::is_thread_cpu_time_supported()) {
  ThreadTotalCPUTimeClosure tttc(CPUTimeCounters::get_instance(), CPUTimeGroups::conc_dedup);
  tttc.do_thread(thread);
}
```

Similar, this can be applied to vmThread.